funnel/jobs/jobs.py

```diff
@@ -0,0 +1,16 @@
+from ..models import (db, TicketClient, SyncTicket)
+from ..extapi.explara import ExplaraAPI
+from flask.ext.rq import job
+from funnel import app
+
+
+@job('talkfunnel')
```

Comment:
Make this queue `funnel` since we're still using `funnel` as the package name for this project (and as the user and database name in production).